pallets/crowdloan-reward/src/lib.rs

```diff
@@ -322,38 +326,26 @@ pub mod pallet {
   ///
   /// It allows to build genesis storage.
   #[pallet::genesis_config]
-	pub struct GenesisConfig {
-    #[doc = "Conversion rate between relay chain and parachain tokens."]
-    pub conversion: u32,
-    #[doc = "Direct reward payout ratio."]
-    pub direct_payout: u32
+  pub struct GenesisConfig {
```

Comment:
Wee can remove these instead of keeping them empty. WDYT ?

Reply:
I thought so too. But the compiler is complaining about a missing `GenesisConfig`. I am currently unsure why. Maybe it is because of the tight coupling with the vesting pallet.

Reply:
ahh cool. if not possible then ls leave it then